### PR TITLE
[WFLY-18487] jta-crash-rec Quickstart CY2023Q3 enhancements

### PR DESCRIPTION
**Pull request by jmfinelli**
Issue [WFLY-18487](https://issues.redhat.com/browse/WFLY-18487)

### CODE REVIEW

File: jta-crash-rec/.github/workflows/quickstart_jta-crash-rec_ci.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: WildFly jta-crash-rec Quickstart CI
```
Comment (kabir) [inline jta-crash-rec/.github/workflows/quickstart_jta-crash-rec_ci.yml:1]: @jmfinelli This is in the wrong folder

Comment (jmfinelli) [reply jta-crash-rec/.github/workflows/quickstart_jta-crash-rec_ci.yml]: Thanks @kabir. I moved it to the right folder (I hope)

**Review by kabir — APPROVED** [review]

**Comment by emmartins** [comment]
thank you @jmfinelli , I created a followup JIRA to have a look at the OpenShift compatibility ([WFLY-18684](https://issues.redhat.com/browse/WFLY-18684))

**Comment by jmfinelli** [comment]
Thanks @emmartins

**Pull request merged**